person.rb

```diff
@@ -32,5 +32,5 @@ def can_use_services?
   end
 end
 
-me = Person.new(14, 'donard')
+me = Person.new(14, 'Isabel')
```

Comment:
- [ ] [REQUIRED] I got an error when I run your APP. Please, make sure that an argument is passed when you use the **correct_name** method.
![1](https://user-images.githubusercontent.com/58552748/144395314-e8b2159d-7256-4873-a3f0-786a702aae05.jpg)

Reply:
Fixed